src/schema/schema.graphql

```diff
@@ -1783,7 +1793,7 @@ input SendMessageInput {
 	id: ID #id
 	origId: ID #origid
 	folderId: ID # l
-	attach: [AttachmentInput]
+	attach: [AttachmentAndDocumentInput]
```

Comment:
let's not rename it, document is also getting passed as attachment only

Reply:
done